catkin_ws/src/image_rotation/src/image_rotation_publisher.cpp

```diff
@@ -0,0 +1,47 @@
+///
+/// @file
+/// @brief Implementation of ROS Publisher
+///
+
+#include <iostream>
+
+#include "../include/image_rotation/image_algorithm.h"
+
+#include <cv_bridge/cv_bridge.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+#include <image_transport/image_transport.h>
+#include <ros/ros.h>
+#include <rosbag/bag.h>
+#include <rosbag/view.h>
+#include "sensor_msgs/Image.h"
+
+sensor_msgs::ImageConstPtr msg_image;
+
+int main(int argc, char** argv)
```

Comment:
use char* argv[] to be consistent with the other implementation

Reply:
done